components/esp_modem/examples/modem_tcp_client/README.md

```diff
@@ -0,0 +1,10 @@
+# Modem TCP client
+
+(See the README.md file in the upper level 'examples' directory for more information about examples.)
+
+## Overview
+This example demonstrates how to act as a MQTT client using modem's TCP commands (provided, the device supports "socket" related commands)
+
+### Supported IDF versions
+
+This example is only supported from `v4.1`, as this is the default dependency of `esp-modem` component.
```

Comment:
It's written only supports v4.1, but here we added other versions?
https://github.com/espressif/esp-protocols/pull/156/files#diff-70666865fb5fbc611086c2fd040c542baa487438cba3a2103e6a3749092c6a26:~:text=example%3A%20modem_console-,%2D%20idf_ver%3A%20%22release%2Dv4.1%22,example%3A%20modem_tcp_client,-%2D%20idf_ver%3A%20%22release%2Dv4.1

Reply:
leftover from the original modem client . will remove